documentation/source/users/rmg/installation/virtualMachineSetup.rst

```diff
@@ -4,6 +4,40 @@
 Setting up a Linux Virtual Machine from Windows
 ************************************************
 
+Checking if Virtualization Technology is Enabled
+=================================================
+
+While most PCs have CPUs that support running virtual machines (i.e. virtualization technology), it is possible that
+this feature has been disabled in your BIOS settings. If this is the case, then you will have to enable this technology
+in your BIOS settings.
+
+First, check to see if virtualization technology is currently enabled on your computer by opening the Task Manager in
+Windows. Click on the ``Performance`` tab--from here you can see if virtualization technology is enabled.
```

Comment:
Does this work for Windows 7?

Reply:
No it does not, only for Windows 8 and later. I have added the appropriate instructions for Windows 7 and earlier.